src/main/kotlin/net/developerden/devdenbot/listener/DiscordTokenMessageListener.kt

```diff
@@ -0,0 +1,53 @@
+package net.developerden.devdenbot.listener
+
+import net.developerden.devdenbot.discord.BOT_COMMANDS_CHANNEL_ID
+import net.developerden.devdenbot.extensions.await
+import net.developerden.devdenbot.serialization.DDBConfig
+import net.developerden.devdenbot.util.handleEachIn
+import net.developerden.devdenbot.util.listenFlow
+import net.developerden.devdenbot.util.scope
+import net.dv8tion.jda.api.JDA
+import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent
+import javax.inject.Inject
+
+class DiscordTokenMessageListener @Inject constructor(
+    private val ddbConfig: DDBConfig,
+    ) : EventListener {
+
+    companion object {
+        private val log by log()
+
+        internal val tokenRegex = Regex("[MN][A-Za-z\\d]{23}\\.[\\w-]{6}\\.[\\w-]{27}\n")
```

Comment:
![image](https://user-images.githubusercontent.com/18754735/132959358-1f71df4b-923e-4f99-84e2-1ea1167363d6.png)

This regex doesn't seem to actually work